packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -904,6 +913,24 @@ export class GarbageCollector implements IGarbageCollector {
 			lastSummaryTime: this.getLastSummaryTimestampMs(),
 			viaHandle: requestHeaders?.[RuntimeHeaders.viaHandle],
 		});
+
+		// We may throw when loading an Inactive object, depending on these preconditions
+		const shouldThrowOnInactiveLoad =
+			!this.isSummarizerClient &&
+			this.configs.throwOnInactiveLoad === true &&
+			requestHeaders?.[AllowInactiveRequestHeaderKey] !== true;
+		const state = this.unreferencedNodesState.get(nodePath)?.state;
+
+		if (shouldThrowOnInactiveLoad && reason === "Loaded" && state === "Inactive") {
+			const request: IRequest = { url: nodePath };
+			const error = responseToException(
```

Comment:
Should we also log when we return this error? We can log extra properties like `packagePath` and incoming `requestHeader` which will tell us which data stores are having issues and whether the request is from a handle (viaHandle request header will be true).

Reply:
We should also consider returning the `packagePath` which may be helpful to the callers.

Reply:
Actually, it will be logged above anyway right, no need to log here.